create-fake-rpm

```diff
@@ -48,14 +48,22 @@ if [ -z "$NAME" -o -z "$PROVIDES" ]; then
   exit 1
 fi
 
-OUTPUT=$( rpmbuild --define="fake_name $NAME" --define="fake_provides $PROVIDES" \
-        --define="fake_version $VERSION" --define="fake_release $RELEASE" \
+SPECFILE=${PWD}/${NAME}.spec
+>${SPECFILE}
```

Comment:
I would rather prefer here `touch ${SPECFILE}` which is more obvious what we are doing here.